Test/test.js

```diff
@@ -7,6 +7,80 @@ const app = require('../server/server.js');
 const should = chai.should();
 chai.use(chaiHttp);
 
+describe('/POST description', () => {
+  it('should POST a song description', (done) => {
+    let description = {
+      "songId": 14,
+      "bandId": 14,
+      "bandName": "sick name toly",
+      "description": "sick description bruhhh"
+    }
+    chai.request(app)
+      .post('/songDescription')
+      .send(description)
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.data.should.be.an('object');
+        res.body.data.songId.should.be.a('number');
+        res.body.data.songId.should.be.equal(14);
+        res.body.data.bandId.should.be.equal(14);
+        res.body.data.description.should.be.a('string');
+        res.body.data.bandName.should.be.a('string');
+        res.body.data.bandName.should.be.equal('sick name toly');
+        done();
+      });
+  });
+});
+
+describe('/PUT descriptions', () => {
+  it('Should update a song description by songId', (done) => {
+    let description = {
+      "bandName": "DOPE name toly",
+      "description": "its aliveeeeee"
+    }
+    chai.request(app)
+      .put('/songDescription/14')
+      .send(description)
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.data.should.be.an('object');
+        res.body.data.songId.should.be.a('number');
+        res.body.data.songId.should.be.equal(14);
+        res.body.data.bandId.should.be.equal(14);
+        res.body.data.description.should.be.a('string');
+        res.body.data.bandName.should.be.a('string');
+        res.body.data.bandName.should.be.equal('DOPE name toly');
+        res.body.data.description.should.be.equal('its aliveeeeee');
+        done();
+      });
+  });
+});
+
+describe('/DELETE descriptions', () => {
```

Comment:
What happens if a song that doesn't exist is queued for deletion? You should confirm that errors are being handled correctly.